src/main/menu.main.ts

```diff
@@ -441,11 +441,20 @@ export class MenuMain extends BaseMenu {
                 (template[template.length - 1].submenu as MenuItemConstructorOptions[]).concat(aboutMenuAdditions);
         }
 
-        (template[template.length - 2].submenu as MenuItemConstructorOptions[]).splice(1, 0, {
-            label: this.main.i18nService.t('hideToTray'),
-            click: () => this.main.messagingService.send('hideToTray'),
-            accelerator: 'CmdOrCtrl+Shift+M',
-        });
+        (template[template.length - 2].submenu as MenuItemConstructorOptions[]).splice(1, 0,
+            {
+                label: this.main.i18nService.t('hideToTray'),
+                click: () => this.main.messagingService.send('hideToTray'),
+                accelerator: 'CmdOrCtrl+Shift+M',
+            },
+            {
+                type: 'checkbox',
+                label: this.main.i18nService.t('alwaysOnTop'),
+                checked: this.windowMain.win.isAlwaysOnTop() ? true : false,
+                click: () => this.main.messagingService.send('alwaysOnTop'),
```

Comment:
Could it just call `this.main.windowMain.alwaysOnTop()` directly instead of sending a message?